keyboards/keebfront/vanguard65/keymaps/via/keymap.c

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2022 <hello@keebfront.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "analog.h"
+#include "string.h"
+
+enum layers {
+    _LAYER0,
+    _LAYER1,
+    _LAYER2,
+    _LAYER3
+};
+
+enum custom_keycodes {
+    A_CW = QK_KB_0,
+    A_CCW
+};
+
+// Set up EEPROM. Mostly boilerplate from QMK docs
+typedef union {
+    uint32_t raw;
+    struct {
+        uint8_t slider_func_state : 8; // Hold only the slider function state. Will be a low value integer, so 8 bits will be used to store.
+    };
+} user_config_t;
+user_config_t user_config; 
+
+uint8_t slider_func = 0;
+
+// On keyboard initialization, pull the EEPROM values
+void keyboard_post_init_user(void) {
+    // Call the keymap level matrix init.
+    user_config.raw = eeconfig_read_user();
+    slider_func = user_config.slider_func_state;
+};
+
+// When the EEPROM gets forcefully reset, set the initialization value
+void eeconfig_init_user(void) {
+    user_config.raw = 0;
+    user_config.slider_func_state = 0;
+    eeconfig_update_user(user_config.raw);
+}
+
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    [_LAYER1] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [_LAYER2] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [_LAYER3] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [_LAYER0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
```

Comment:
```suggestion
    [_LAYER0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
    [_LAYER1] = { ENCODER_CCW_CW(_______, _______) }
```

Reply:
@waffle87 Same reason as above, I cannot commit this without breaking the core functionality of the slider and making it difficult to add more layers.